services/kg-core-api/src/main/java/eu/ebrains/kg/core/controller/CoreInstanceController.java

```diff
@@ -311,7 +322,7 @@ public Paginated<NormalizedJsonLd> getIncomingLinks(UUID id, DataStage stage, St
             return null;
         }
         final SpaceName privateSpaceName = authContext.getUserWithRoles().getPrivateSpace();
-        final Paginated<NormalizedJsonLd> incomingLinks = graphDBInstances.getIncomingLinks(instanceId.getSpace().getName(), instanceId.getUuid(), stage, property, type.getName(), pagination);
+        final Paginated<NormalizedJsonLd> incomingLinks = graphDBInstances.getIncomingLinks(instanceId.getSpace().getName(), instanceId.getUuid(), stage, property != null ? property : null, type != null ? type.getName() : null, pagination);
```

Comment:
what is the property!=null ? property : null check for? I do get the type null-check but for the property it doesn't seem to do anything

Reply:
It was for testing in local. I pushed it by mistake. solve now